lib/Module/Build.pm

```diff
@@ -982,71 +981,54 @@ ExtUtils::MakeMaker> for further information.
 =back
 
 
-=head1 MOTIVATIONS
+=head1 COMPARISON
 
-There are several reasons I wanted to start over, and not just fix
-what I didn't like about C<MakeMaker>:
+A comparison between C<Module::Build> and other CPAN distribution installers.
 
-=over 4
+=over
 
 =item *
 
-I don't like the core idea of C<MakeMaker>, namely that C<make> should be
-involved in the build process.  Here are my reasons:
-
-=over 4
-
-=item +
-
-When a person is installing a Perl module, what can you assume about
-their environment?  Can you assume they have C<make>?  No, but you can
-assume they have some version of Perl.
-
-=item +
-
-When a person is writing a Perl module for intended distribution, can
-you assume that they know how to build a Makefile, so they can
-customize their build process?  No, but you can assume they know Perl,
-and could customize that way.
-
-=back
-
-For years, these things have been a barrier to people getting the
-build/install process to do what they want.
+L<ExtUtils::MakeMaker> requires C<make> and use of a C<Makefile>.
+C<Module::Build> does not, nor do other pure-perl installers following the
+C<Build.PL> spec such as L<Module::Build::Tiny>. In practice, this is usually
```

Comment:
Should use `F<...>` for files.

It would be nice to link to [`CPAN::API::BuildPL`](https://github.com/Perl-Toolchain-Gang/cpan-api-buildpl/blob/master/lib/CPAN/API/BuildPL.pm) eventually, but it is unfinished and unpublished on CPAN.